Resources/Prototypes/Recipes/Construction/Graphs/furniture/tables.yml

```diff
@@ -63,6 +63,18 @@
               amount: 1
               doAfter: 1
 
+        - to: CounterWood
+          steps:
+            - material: WoodPlank
+              amount: 1
+              doAfter: 1
+              
+        - to: CounterMetal
+          steps:
+            - material: Steel
+              amount: 1
+              doAfter: 1
```

Comment:
...doesnt this conflict with the other edges that do the same thing?

Reply:
Correct, was an issue I ran into even though you select a different recipe from the menu it wasn't playing ball, even if I added a different tool step the default one takes priortiy *shrug*